doc/devguide/contributing.rst

```diff
@@ -88,7 +88,7 @@ Pull Requests
 
 Please use the provided pull request template. In particular:
 
--  **Make sure the test suite passes** on your computer. To do so, run ``py.test saltproc`` in the repository directory. At a minumum, you must run the tests requring serpent locally as they are not tested by our CI
+-  **Make sure the test suite passes** on your computer. To do so, run ``py.test saltproc`` in the repository directory. At a minimum, you must run the tests reqiuring serpent locally as they are not tested by our CI
```

Comment:
Because the official pytest docs use `pytest` as the recommended command, we may want to revise this to be in line with them. 
```suggestion
-  **Make sure the test suite passes** on your computer. To do so, run ``pytest saltproc`` in the base directory of this repository. At a minimum, you must run the tests requiring serpent locally as they are not tested by our CI
```

Reply:
Good catch